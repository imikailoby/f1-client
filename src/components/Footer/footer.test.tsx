/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react'
import { Footer } from './';

describe('<Footer />', () => {
  it('renders without crashing', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot();
  });
});
