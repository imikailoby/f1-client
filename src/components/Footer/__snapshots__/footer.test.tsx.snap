// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders without crashing 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      F1 Client Â© 2022
    </p>
  </footer>
</div>
`;
